### PR TITLE
Add discord links

### PR DESCRIPTION
**Pull request by gagb**
This pull request includes a small but important change to the `python/packages/autogen-core/docs/src/conf.py` file. The change adds a new social media link to the documentation configuration.

* Added a new Discord link to the social media section in the documentation configuration file `conf.py`.

image

### CODE REVIEW

File: python/packages/autogen-core/docs/src/conf.py [hunk 1]
```diff
@@ -103,6 +103,11 @@
             "url": "https://twitter.com/pyautogen",
             "icon": "fa-brands fa-twitter",
         },
+        {
```
Comment (victordibia) [inline python/packages/autogen-core/docs/src/conf.py:106]: LGTM.
Can you add a screenshot of what this looks like, before, after?
Mostly to gauge space utilization

Comment (gagb) [reply python/packages/autogen-core/docs/src/conf.py:106]: added, see pr description @victordibia. should we remove the pypi icon? its redundant with the packages link in the left?

**Review by jackgerrits — APPROVED** [review]

File: python/packages/autogen-core/docs/src/conf.py (hunk 1 above)
Comment (victordibia) [reply python/packages/autogen-core/docs/src/conf.py:106]: Yes. I vote to remove the pypi icon. Thanks!

**Review by victordibia — APPROVED** [review]

File: python/packages/autogen-core/docs/src/conf.py (hunk 1 above)
Comment (rysweet) [reply python/packages/autogen-core/docs/src/conf.py:106]: image

Comment (rysweet) [reply python/packages/autogen-core/docs/src/conf.py:106]: before

**Review by ekzhu — APPROVED** [review]

File: python/packages/autogen-core/docs/src/conf.py (hunk 1 above)
Comment (afourney) [reply python/packages/autogen-core/docs/src/conf.py:106]: Is this resolved? Are we good to go?

**Pull request merged**